lib/utils/checkInvalidCLIOptions.js

```diff
@@ -72,10 +72,10 @@ const buildMessageLine = (invalid, suggestion) => {
  * @return {string}
  */
 module.exports = function checkInvalidCLIOptions(allowedOptions, inputOptions) {
-	const allOptions = buildAllowedOptions(allowedOptions);
+	const allOptions = buildAllowedOptions(allowedOptions).map(_.kebabCase);
 
 	return Object.keys(inputOptions)
```

Comment:
Is it possible to `filter` at first? It seems that in this case there would be no need in `map` at `allOptions` const... I might be wrong :)

Reply:
@fanich37 You're right! Thank you. I fixed via 9c9d363116c2a9b7247e5455b33a2a7fda3242f2.